.github/workflows/draft-release.yml

```diff
@@ -92,22 +92,24 @@ jobs:
           cosignPassword: ${{ inputs.key == 'release' && secrets.COSIGN_PASSWORD || secrets.COSIGN_DEV_PASSWORD }}
 
       - name: Upload CLI as artifact (unix)
-        uses: actions/upload-artifact@a8a3f3ad30e3422c9c7b888a15615d19a852ae32 # v3.1.3
+        uses: ./.github/actions/artifact_upload
```

Comment:
nit: We don't technically need to encrypt these artifacts since they'll be published as part of the release anyways